prow/config/jobs/istio.io.yaml

```diff
@@ -6,3 +6,31 @@ image: gcr.io/istio-testing/build-tools:master-2020-02-14T13-09-14
 jobs:
   - name: lint
     command: [make, lint]
+
+  - name: k8s-tests
+    type: presubmit
+    command: [entrypoint, prow/integ-suite-kind.sh, test.kube.presubmit]
+    requirements: [kind]
+    modifiers: [optional, hidden]
+
+  - name: k8s-tests
+    type: postsubmit
+    command: [entrypoint, prow/integ-suite-kind.sh, test.kube.postsubmit]
+    requirements: [kind, gcp]
+    modifiers: [optional, hidden]
+
+resources:
+  default:
+    requests:
+      memory: "3Gi"
+      cpu: "5000m"
+    limits:
+      memory: "24Gi"
+      cpu: "8000m"
+  lint:
```

Comment:
this isn't used, since no job references `resources: lint`. So only the default mentioned above will apply to the two new jobs you added as well as lint. Seems reasonable to have just the default